Content.Shared/Movement/Systems/SharedMoverController.Input.cs

```diff
@@ -21,6 +22,11 @@ public abstract partial class SharedMoverController
     {
         public bool CameraRotationLocked { get; set; }
 
+        [Dependency] private readonly AlertsSystem _alerts = default!;
+
+        [ValidatePrototypeId<AlertPrototype>]
+        public const string WalkingAlert = "Walking";
```

Comment:
```suggestion
        public const ProtoId<AlertPrototype> WalkingAlert = "Walking";
```
ProtoId should be used instead of ValidateProtoId
![grafik](https://github.com/user-attachments/assets/a584b277-4821-443a-aa17-390b05b44455)